### ISSUE TITLE
White screen with 0 configuration webpack plugin

### ISSUE THREAD

**Issue by pivaszbs** [statoscope/statoscope#109]
**Describe the bug**
Nothing is visible on the screen and errors in the console

**To Reproduce**
Steps to reproduce the behavior: idk, I just install webpack plugin w/o configuration :)

**Expected behavior**
Stats on the screen

**Screenshots**
![image](https://user-images.githubusercontent.com/26435684/134716540-f3d97dd7-ca19-467b-a0be-c52f1a6548b5.png)

**Versions**:
 - Node (for non-browser bug):  12
 - Browser (for browser-bug): Yandex Browser 92.0.4515.1716
 - Statoscope: "@statoscope/webpack-plugin": "5.8.0"

**Comment by smelukov** [comment]
@pivaszbs please, provide some additional info:
webpack version
`stats` field from your webpack config
seems like you have webpack 5 and `all: true` in your stats config )

**Comment by pivaszbs** [comment]
@smelukov webpack: 5
zero config as I said :)

**Comment by smelukov** [comment]
Please, try to use plugin with this configuration:
```js
new StatoscopeWebpackPlugin({
  statsOptions: {
    all: false, // disable all the stats
    hash: true, // add compilation hash
    entrypoints: true, // add entrypoints stats
    chunks: true, // add chunks stats
    chunkModules: true, // add modules stats
    reasons: true, // add modules reasons stats
  
    assets: true, // add assets stats
    chunkOrigins: true, // add chunks origins stats (to find out which modules require a chunk)
    version: true, // add webpack version
    builtAt: true, // add build at time
    timings: true, // add build at time
    performance: true, // add info about oversized assets
    source: true, // add module sources (uses to find modules duplicates)
  },
  open: 'file',
});
```
Is it fix your problem?

**Comment by pivaszbs** [comment]
Actually crashed with
    `TypeError: resolvedModule.chunks is not iterable at handleModule (<project_path>/node_modules/@statoscope/webpack-model/dist/normalizeCompilation.js:12:64) at handleChunk (<project_path>/node_modules/@statoscope/webpack-model/dist/normalizeCompilation.js:19:29) at handleCompilation (<project_path>/node_modules/@statoscope/webpack-model/dist/normalizeCompilation.js:36:9) at Object.normalizeCompilation [as default] (<project_path>/node_modules/@statoscope/webpack-model/dist/normalizeCompilation.js:44:9) at <project_path>/node_modules/@statoscope/webpack-plugin/dist/index.js:73:43`
    
 Attempt to remove `chunks` option not crash build and lead to this behavior
![image](https://user-images.githubusercontent.com/26435684/134719357-09ef619a-f282-4293-b55d-9a1b2902a1bb.png)

**Comment by smelukov** [comment]
@pivaszbs thanks for the issue 🙏
It will be fixed in the near release

**Comment by smelukov** [comment]
Fixed in v5.9
